pom.xml

```diff
@@ -103,11 +105,11 @@
     <distributionManagement>
```

Comment:
Do we still need this section in the POM?

Reply:
I think we do, We are not using Ripe parent pom.xml. Also we are using maven command to upload and download artifacts from nexus.  where else do you think this information should come from?

Reply:
OK! Just wanted to confirm that we need the info in the POM.